src/api/mod.rs

```diff
@@ -147,7 +147,7 @@ pub struct PeripheralProperties {
     /// The address of this peripheral
     pub address: BDAddr,
     /// The type of address (either random or public)
-    pub address_type: AddressType,
+    pub address_type: Option<AddressType>,
```

Comment:
Having an address without an `AddressType` seems problematic, isn't it required to know how to interpret the address?

Reply:
The problem is that not all Bluetooth systems report the address type. Android does not, and in fact, neither do WinRTBLE and CoreBluetooth - out of the systems we use, the only one that does is BlueZ. Rather than pretending to know the type and guessing, BTLEPlug should simply be honest that it doesn't know, and let the client decide what to do with that information.

I've updated WinRTBLE and CoreBluetooth to reflect this.